packages/node_modules/@node-red/nodes/core/common/20-inject.html

```diff
@@ -483,12 +586,7 @@
                 if (this.changed) {
                     return RED.notify(RED._("notification.warning", {message:RED._("notification.warnings.undeployedChanges")}),"warning");
                 }
-                var payload = this.payload;
```

Comment:
I removed this because the payload variable wasn't used. Seems like left over code.